src/main/frontend/handler/page.cljs

```diff
@@ -428,7 +428,7 @@
   []
   (let [current-length (:journals-length @state/state)]
     (when (< current-length (db/get-journals-length))
-      (state/update-state! :journals-length inc))))
+      (state/set-journals-length! (+ current-length 10)))))
```

Comment:
The default length of journals can be 10 but it seems incrementally load another 10 after scrolling is a bit too many.

Reply:
Yes, in my testing loading 10 notes is not as fast as I thought, let me turn it down a bit. Also, I'm trying to build a new load strategy based on IntersectionObserver. Will ping you when it's ready.

Reply:
How about loading the first two journals (both today and yesterday), we don't want to load too many journals because it will be slower to load if there're many notes, and it'll be slower to edit if there're some queries too.

Reply:
I think it is not so necessary to really solve the issue perfectly or aggressively load more journals than a user normally uses.
From another angle of view, we could have a more UX-friendly way to let the users know there are more journals under the current view, e.g., having a "Load More Journals Button" or indicator below the end of the last journal.

Reply:
@pengx17 agreed!